include/mbgl/renderer/layer_tweaker.hpp

```diff
@@ -93,9 +93,12 @@ class LayerTweaker {
     std::vector<std::string> propertiesAsUniforms;
 #endif // MLN_RENDER_BACKEND_METAL
 
+    // Indicates that the evaluated properties have changed
     bool propertiesUpdated = true;
 
-    bool propertiesChanged = true;
```

Comment:
`propertiesChanged` is still referenced. I get a compile error in `LineLayerTweaker` / metal.

Reply:
Forgot to try Metal, sorry.